app/usecases/performance.py

```diff
@@ -65,16 +65,8 @@ def calculate_performances(
     results: list[PerformanceResult] = []
 
     for score in scores:
-        # assert either acc OR 300/100/50/geki/katu/miss is present, but not both
-        # if (score.acc is None) == (
-        #     score.n300 is None
-        #     and score.n100 is None
-        #     and score.n50 is None
-        #     and score.ngeki is None
-        #     and score.nkatu is None
-        #     and score.nmiss is None
-        # ):
-        #     raise ValueError("Either acc OR 300/100/50/geki/katu/miss must be present")
+        if score.acc and (score.ngeki or score.nkatu or score.n50 or score.n100):
```

Comment:
n300?

Reply:
When does it make sense to specify n300? With score.acc it's estimated, and when u specify 100s etc. n300 is just the leftover.

Reply:
Yeah I agree - I think we should _block_ usage of `n300` and `acc` together, the same as with geki/katu/n50/n100.

More specifically, outside of misses, any fields which affect `acc`:

![image](https://i.cmyui.xyz/UqlHsVoUcoskniD2QVGt.png)

Reply:
I think we can just remove the n300 param completely? I dont think it ever has a usecase